lib/menu.c

```diff
@@ -360,6 +360,48 @@ bm_menu_get_border_size(struct bm_menu* menu)
     return menu->border_size;
 }
 
+void
+bm_menu_set_border_radius(struct bm_menu* menu, uint32_t border_radius)
+{
+    assert(menu);
+    menu->border_radius = border_radius;
+}
+
+uint32_t
+bm_menu_get_border_radius(struct bm_menu* menu)
+{
+    assert(menu);
+    return menu->border_radius;
+}
+
+uint32_t
```

Comment:
these don't need to be a public API, simply store this information in the menu structs and write / read them directly.

Reply:
Also you don't need any of these functions or data in the menu struct, as you can get count of filtered items with `menu->filtered.count` and `menu->items.count`

Reply:
I was thinking that I was a bit overcomplicating it for the sake of standardization, I'll look into fixing the code.

Reply:
`menu->items.count` works great, but `menu->filtered.count` shows **0** when nothing is typed in. While **0** makes sense logically here as nothing is filtered yet, the requested feature showed "totalitemcount/totalitemcount" if nothing was typed in. I'll most likely have to add some things in the `bm_menu_filter` function(I believe that's the name) to fix this, which I hope is ok with you.

Reply:
Just committed a fix for both the issue with 0 and the over complication, I'll look into re-basing it.

Reply:
I think you can simply use `menu->items.count` if `menu->filter` is null